linters/js/README.md

```diff
@@ -0,0 +1,11 @@
+ESLint Setup
+============
+
+1. Copy [.eslintrc](.eslintrc) to the root of the project.
+2. Install `eslint-config-airbnb`.
+
+
+    npm i --save-dev eslint-config-airbnb
```

Comment:
I think that conversations needs to happen, but not here. At the moment, some people are experimenting with `yarn`, but our default remains `npm`.